src/libraries/System.Collections.Immutable/tests/ImmutableArrayTest.netstandard21.cs

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Nit: tests projects never target netstandard, please use `netcoreapp` instead per suggestion in the other projects.

Reply:
Fixed.